README.md

```diff
@@ -0,0 +1,59 @@
+# ra-keycloak
+
+An auth provider for [react-admin](https://github.com/marmelab/react-admin) which handles authentication via a [Keycloak](https://www.keycloak.org/guides) server.
+
+This repository contains:
+- The actual `ra-keycloak` package
+- A simple demo app you can run locally to try out `ra-keycloak`
+
+## The `ra-keycloak` package
+
+- Please have a look at the [DOCUMENTATION](./packages/ra-keycloak/Readme.md)
+- And also why not the [source code](https://github.com/marmelab/ra-keycloak/tree/main/src/packages/ra-keycloak)
+
+## Simple Demo
+
+### Prerequesites
+
+This demo requires **Docker** and **docker-compose** in order to start a local Keycloak server.
+
+### Initial setup
+
+1. Clone this project
+1. Run `make install run` to install the dependencies, start the local Keycloak Server and start the Demo App
+
+We need to add some minimal configuration to our Keycloak server in order to use it. This need to be done from the Keycloak Admin Console.
+
+1. Browse to http://localhost:8080/ (note: keycloak takes some time to start...)
+1. Go to **Administration Console**
+1. Login with the default credentials (admin / password)
+1. Create a new **Realm** named for instance `Marmelab`
+1. Create a new **Realm Role** named `admin`
+1. Create a new **Realm Role** named `user`
+1. Create a new **User** named `admin@marmelab.com`
+1. For `admin@marmelab.com`, under **Credentials**, set a new password (disable the temporary password tick)
+1. For `admin@marmelab.com`, under **Role mapping**, click **Assign role** and choose `admin`
+1. Create a new **User** named `user@marmelab.com`
+1. For `user@marmelab.com`, under **Credentials**, set a new password (disable the temporary password tick)
+1. For `user@marmelab.com`, under **Role mapping**, click **Assign role** and choose `user`
+1. Create a new **Client** and choose as **Client ID** `front-marmelab`. Leave all the other options to default.
+1. For Client `front-marmelab`, under **Settings**, edit the **Access settings** to the following:
+  - Root URL: http://localhost:8081/
+  - Home URL: http://localhost:8081/
+  - Valid redirect URIs: http://localhost:8081/*
+  - Valid post logout redirect URIs: http://localhost:8081/*
+  - Web origins: *
+15. Click **Save**. Your keycloak instance is fully configured.
```

Comment:
```suggestion
17. Click **Save**. Your keycloak instance is fully configured.
```

Reply:
Thanks for the review.

Actually I believe it is valid markdown to start an ordered list with only `1.`. Have a look at the resulting file in the Github markdown viewer: https://github.com/marmelab/ra-keycloak/blob/019c9f47790c98bfcc90c11509183570078bbaad/README.md

Only exception being the last line, for which I needed to hardcode the line number, because the markdown interpreter gets confused because I embedded an unordered list in the ordered list.